src/response_handler.js

```diff
@@ -52,11 +52,11 @@ export function handle_explore_response(query, response) {
   if (query === 'tagset') {
     return getTagset(response);
   }
-  else if (query === 'apps') {
-    return getTagset(response)['App'];
+  else if (query === 'cluster') {
```

Comment:
Yes we are moving to new version, instead of “APP” we call it “Cluster”, which is more appropriate.

~ Subhash


From: Alexander Zobnin <notifications@github.com>
Reply-To: Staples-Inc/mqe-datasource <reply@reply.github.com>
Date: Friday, September 9, 2016 at 9:54 AM
To: Staples-Inc/mqe-datasource <mqe-datasource@noreply.github.com>
Subject: Re: [Staples-Inc/mqe-datasource] Incorporated Alias indexing for metrics containing Wildcards (#7)


In src/response_handler.js<https://github.com/Staples-Inc/mqe-datasource/pull/7#discussion_r78183451>:

> @@ -52,11 +52,11 @@ export function handle_explore_response(query, response) {

>    if (query === 'tagset') {

>      return getTagset(response);

>    }

> -  else if (query === 'apps') {

> -    return getTagset(response)['App'];

> +  else if (query === 'cluster') {

Do you use another API version? My version returns object like this:

success: true

body: {

  functions: [...],

  metrics: [...],

  tagset: [

    {

      App: "demoapp",

      Hosts: ["staples-lab-1", "staples-lab-2"]

    }

  ]

}

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/Staples-Inc/mqe-datasource/pull/7/files/e298294f3a1415d47b6d632deecc3c30924d4a78..17750446945757f8f3035d5982d2d8aa2a8601e9#r78183451>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ATCO5CS9A7C33ywUsxvh8FOXPHyPOv1Jks5qoWUZgaJpZM4J3OT3>.